controllers/managedocs_controller.go

```diff
@@ -1228,8 +1228,16 @@ func (r *ManagedOCSReconciler) reconcileOCSCSV() error {
 		return fmt.Errorf("OCS CSV not found")
 	}
 	var isChanged bool
+	var _0 = int32(0)
 	deployments := csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs
 	for i := range deployments {
+		deployment := &deployments[i]
+		// Disable noobaa operator by scaling down the replica of noobaa deploymnet 
+		// in OCS CSV.
+		if deployment.Name == "noobaa-operator" && deployment.Spec.Replicas != &_0 {
```

Comment:
`deployment.Spec.Replicas != &_0` -> You are checking if the point has the same address of the local variable which will always be false.

The proper check should be:
```suggestion
		if deployment.Name == "noobaa-operator" && 
		(deployment.Spec.Replicas == nil |  *deployment.Spec.Replicas > 0) {
```